scripts/build-configuration-wiki.js

```diff
@@ -3,9 +3,8 @@ const fs = require('fs');
 const pkg = require('../package');
 const schema = require('../settings-schema');
 const properties = schema.properties;
-const settings = require('../src/configuration/rendererSettings.js').default;
-const chartSettings = require('../src/configuration/chartSettings.js').default;
-const listingSettings = require('../src/configuration/listingSettings.js').default;
+const rendererSettings = require('../src/configuration/rendererSettings.js').default;
+const webchartsSettings = require('../src/configuration/webchartsSettings.js').default();
```

Comment:
is default a method?

Reply:
i think that's how babel interprets import/export syntax in the context of require().  in this case the settings files return a default function.